### PR TITLE
Minor OATH doc edits

### PR DESCRIPTION
**Pull request by elukewalker**

### CODE REVIEW

File: oath/README.md [hunk 1]
```diff
@@ -81,10 +87,13 @@ Authenticators often use QR codes to pass the URL for setting up the credentials
...
+2. Select "OATH demo" pivot in navigation drawer
+3. Plug in YubiKey and click + FAB button to add credential.  
+   To test, use one of the services that provides QR codes and authentication with TOTP as 2nd factor auth.  
+   For example, [https://demo.yubico.com/playground](https://demo.yubico.com/playground)  
+   Or such services as Facebook, Google, Amazon, Microsoft, etc. All provide 2nd factor authentication with Authenticator app. This demo can be used as such.
 
 ## Additional Resources <a name="additional_resources"></a>
-OATH - [YK OATH protocol](https://developers.yubico.com/OATH/YKOATH_Protocol.html)
+* [YKOATH Protocol Specification](https://developers.yubico.com/OATH/YKOATH_Protocol.html)
+* [Yubico Authenticator for Android open source project](https://github.com/Yubico/yubioath-android)
```
Comment (imakhalova) [inline oath/README.md:99]: This authenticator doesn't use YubiKit. So I'm not sure if we actually should mention it here.
Demo shows implementation using YubiKit.

Comment (elukewalker) [reply oath/README.md]: Removed. Also checked in some FIDO doc edits

File: otp/README.md [hunk 2]
```diff
@@ -58,8 +62,10 @@ yubikitVersion=1.0.0-beta05
...
 ```
 
-If this dialog does not meet your requirements you can check implementation of OtpActivity class and implement it using **YubiKit** core methods to detect NFC tag or USB connection.
+Note: You can create a custom dialog to meet your UX requirements. Review the implementation of OtpActivity class and detect the NFC tag or USB connection using **YubiKit** core methods.
 
 ### Using the Demo Application <a name="using_demo"></a>
-Run demo app, select "OTP demo" pivot in navigation drawer, tap "Read OTP" button and follow instructions on screen.
-To validate retrieved OTP tap on "Validate OTP" button 
+1. Run demo app
+2. elect "OTP demo" pivot in navigation drawer
```
Comment (imakhalova) [inline otp/README.md:69]: Capitalize E

File: piv/README.md [hunk 3]
```diff
@@ -47,5 +45,11 @@ And use it to create `PivApplication` to select PIV applet on YubiKey
 ```
 
 ### Using the Demo Application <a name="using_demo"></a>
-Run demo app, select "PIV demo" pivot in navigation drawer. Plug in YubiKey and check what's current certificates you've got. You can generate new key and sign data with that key.
-Current demo doesn't allow you to import certificate from file, but it emulated import from pre-defined asset file. And exports to local cache file. Which also can be used to import certificate into another slot.
\ No newline at end of file
+1. Run demo app
+1. Select "PIV demo" pivot in navigation drawer
```
Comment (imakhalova) [inline piv/README.md:49]: Enumeration is wrong

File: oath/README.md [hunk 4]
```diff
@@ -54,8 +55,13 @@ And use it to create `OathApplication` to select OATH applet on YubiKey
...
-**OATH YubiKit** also provides a class for defining an OATH `Credential`, which has a convenience initializer `Credential.parseUri` which can parse the credential parameters from Uri of [Key Uri Format](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).
+Next, use the `OathApplication` to add, remove, list, and calculate credentials.
+
+The `OathApplication` implements the YKOATH protocol. Refer to the [YKOATH protocol specification](https://developers.yubico.com/OATH/YKOATH_Protocol.html) for more details. 
+
+In addition, the `OathApplication` provides an interface for setting/validating a password on credential storage, calculating all credentials and resetting the OATH application to its default state. For the complete list of methods look at the [`OathApplication` class documentation](src/main/java/com/yubico/yubikit/oath/OathApplication.java).  
+
+The **OATH** module also provides a class for defining an OATH `Credential`, which has a convenience initializer `Credential.parseUri` which can parse the credential parameters from Uri of [Key Uri Format](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).
```
Comment (imakhalova) [inline oath/README.md:64]: May be rephrase it somehow to avoid double which?

Comment (elukewalker) [reply oath/README.md]: fixed in f138d2e

File: piv/README.md (hunk 3 above)
Comment (elukewalker) [reply piv/README.md:49]: Markdown's numbered list feature automatically displays the correct numbering when using '1.'.  See https://guides.github.com/features/mastering-markdown/#syntax

Does this feature resolve the issue?

Comment (imakhalova) [reply piv/README.md:49]: Yes, thank you

**Pull request merged**